src/main/java/core/basesyntax/MakerStatistic.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public class MakerStatistic {
+    private static final String[] NAME_FIELDS = {"supply", "buy", "result"};
+    private static final int INDEX_SUPPLY = 0;
+    private static final int INDEX_BUY = 1;
+    private static final int INDEX_RECORD = 2;
+    private static final char SPACER = ',';
+    private static final int INDEX_NAME_IN_RECORD = 0;
+    private static final int INDEX_AMOUNT_IN_RECORD = 1;
+    private StringBuilder stringBuilder = new StringBuilder();
```

Comment:
This field must be at the method level. Otherwise you have to clean up this value after getStatictic method finished its work